### PR TITLE
refactor oppskrift footer

### PR DESCRIPTION
**Pull request by tnlitv**
#### Please tick a box ###
- [ ] **feature** _(A new feature_)
- [ ] **fix** _(A bug fix_)
- [x] **refactor** _(A code change that neither fixes a bug nor adds a feature_)

#### If you're adding a feature, is it documented in a storybook story?
- [ ] Yes
- [ ] No

#### Are the components you're working on reusable between brands?
- [ ] Yes _(Using variables that are defined in the default theme)_
- [ ] No _(I hard coded some values)_

#### If you're creating markup, did you add proper semantics? 
- [ ] I added [ARIA attributes](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA)
- [ ] I added microdata from [Schema.org](http://schema.org/)
- [ ] Semantics are derived from HTML
- [ ] Not applicable

_(Did you do a CR and see that there is something that we should check for each PR, that are not on the list, please [update this document](https://github.com/dbmedialab/shiny/edit/master/.github/PULL_REQUEST_TEMPLATE.md))_

### CODE REVIEW

File: src/molecules/Footers/OppskriftFooter.js [hunk 1]
```diff
@@ -5,46 +5,31 @@ import { Col, Row, Grid } from '../..';
...
-			margin-bottom: 3rem;
-		}
-		.margin-bottom-sm-6{
-			margin-bottom: 3rem;
-		}
-		.margin-bottom-sm-10{
-			margin-bottom: 3rem;
-		}
+	font-size: ${props => props.theme.variables.uiRegularSize};
+	line-height: ${props => props.theme.variables.headingMediumLineHeight};
```
Comment (misund) [inline src/molecules/Footers/OppskriftFooter.js]: the similarly named fontsizes and line-heights should go pretty well together. uiRegularSize should probably go with uiRegularLineHeight, and headingMediumLineHeight should probably go with headingMediumSize.

Is there anything special about this component that says we should deviate from that pattern?

Comment (tnlitv) [reply src/molecules/Footers/OppskriftFooter.js]: i tried uiRegularLineHeight at first, though it looks better with headingMediumLineHeight (according to designs)

Comment (misund) [reply src/molecules/Footers/OppskriftFooter.js]: How about using headingSmallSize and headingSmallLineHeight?

If the font-size and line-height pair do not match as expected, we should look into if the values should be adjusted.

File: src/molecules/Footers/OppskriftFooter.js [hunk 2]
```diff
@@ -5,46 +5,31 @@ import { Col, Row, Grid } from '../..';
...
+		padding: 10rem 0 6rem;
 	}
 	
 	.text-left{
 		text-align: left;
 	}
 `;
 
+const LeftAlignedCol = Col.extend`
+	text-align: left;
```
Comment (misund) [inline src/molecules/Footers/OppskriftFooter.js]: Isn't this the initial value?

Comment (tnlitv) [reply src/molecules/Footers/OppskriftFooter.js]: nope, it was centered by default for some reason
but that's a good point, i'm going to investigate it

File: src/molecules/Footers/OppskriftFooter.js [hunk 3]
```diff
@@ -58,15 +43,12 @@ const OppskriftLink = styled.a`
 	text-decoration: none;
 	color: inherit;
 	font-family: 'Ubuntu', sans-serif;
+	font-size: ${props => props.theme.variables.headingLargeSize};
```
Comment (misund) [inline src/molecules/Footers/OppskriftFooter.js]: When specifying the font-size, we should probably specify a line-height as well. for a headingLargeSize, I would suggest using headingLargeLineHeight

File: src/molecules/Footers/OppskriftFooter.js (hunk 1 above)
Comment (tnlitv) [reply src/molecules/Footers/OppskriftFooter.js]: hmm, i'm refactoring recipe page right now, gonna try different line height for uiRegular 👍

File: src/molecules/Footers/OppskriftFooter.js (hunk 2 above)
Comment (tnlitv) [reply src/molecules/Footers/OppskriftFooter.js]: for some reason, when specifying `<Row center="md">`, text inside `Col` is also centered. Not sure if this is desired behavior, but anyway

**Pull request merged**